config/initializers/indexing_strategies.rb

```diff
@@ -0,0 +1,7 @@
+ACTIVE_INDEXING_STRATEGIES = [
+  Search::BookVersions::I1::IndexingStrategy
```

Comment:
could put the strings here instead of classes, so don't have to say `.new.version` above which may not be same for all strategies